src/data/screenshots_de.json

```diff
@@ -144,7 +157,7 @@
                 ]
             },
             {
-                "title": "Testergebnis abrufen",
+                "title": "Testergebnis",
                 "anchor": "android_results",
                 "description": "In diesem Abschnitt finden Sie Screenshots zu “Testergebnis abrufen”. Auf der Status-Seite haben sie, sofern Sie einen PCR-Test registriert haben, die Möglichkeit das Testergebnis abzurufen. Es gibt 5 Möglichkeiten. Das Ergebnis liegt noch nicht vor, Negativ, Positiv, nicht mehr gültig und Ihr PCR-Test war fehlerhaft.",
```

Comment:
"Auf der Status-Seite haben sie, sofern Sie einen PCR-Test registriert haben, die Möglichkeit das Testergebnis abzurufen."

"sie" should be "Sie" but anyway the sentence could be made a bit simpler:

"Sofern Sie einen PCR-Test registriert haben, haben Sie die Möglichkeit das Testergebnis auf der Status-Seite abzurufen."

Reply:
Both issues were addressed in https://github.com/corona-warn-app/cwa-website/pull/3005/commits/9c927421c18ff9ba88e1ff3dc681205fdda5dc2c, thanks for the information